core/arch/opnd.h

```diff
@@ -2843,7 +2843,31 @@ DR_API
 bool
 reg_is_stolen(reg_id_t reg);
 
-/* internal version */
+/* Internal version */
+
+#ifdef X86
+
+/* Helper function to retrieve the "input size" as meant in Intel's Spec. Vol.2A "2.6.5
+ * Compressed Displacement (disp8*N) Support in EVEX". This is not a general function
+ * to determine the input size, and shall only be used for the purpose of table mentioned
+ * above.
+ */
+opnd_size_t
+instr_get_input_size_from_opcode(int opcode, bool evexw);
+
+/* Helper function to determine the vector length based on EVEX.L, EVEX.L'. */
+opnd_size_t
+instr_get_vector_length(bool vex_l, bool evex_ll);
```

Comment:
There is no instr_t input so I would say s/instr_/decode_/

Reply:
Renamed.